datadog_lambda/tracing.py

```diff
@@ -328,10 +317,9 @@ def _extract_context_from_eventbridge_sqs_event(event):
 
             detail = body.get("detail")
             dd_context = detail.get("_datadog")
-            trace_id = dd_context.get(TraceHeader.TRACE_ID)
-            parent_id = dd_context.get(TraceHeader.PARENT_ID)
-            sampling_priority = dd_context.get(TraceHeader.SAMPLING_PRIORITY)
-            return trace_id, parent_id, sampling_priority
+            context = propagator.extract(dd_context)
+        if context is not None:
+            return context
     except Exception:
         raise
```

Comment:
This is completely redundant and unneccesary.

Reply:
right, since the original call to this func happens in the try inside extract_context_from_sqs_or_sns_event_or_context